app/models/popular_links_edition.rb

```diff
@@ -34,7 +34,8 @@ def create_draft_popular_links_from_last_record
     popular_links
   end
 
-  def publish
+  def publish_latest
+    save_draft
```

Comment:
This change doesn't seem related to the PR text, and there's no explanation in the commit about why we're doing this—would be good to have the reasoning explained in the commit text (and it should probably be a separate commit).